CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Features
 
-- ASAB WebUI submodule version bump [[ae64c84](https://github.com/TeskaLabs/asab-webui/commit/ae64c84a78ecbbc32a1a02c5bf3bc196907b0263)] commit (INDIGO Sprint 230526, [!32](https://github.com/TeskaLabs/seacat-auth-webui/pull/32))
+- ASAB WebUI submodule version bump [[297aa81](https://github.com/TeskaLabs/asab-webui/commit/297aa819e97964aec33e03f545fc10819f0fe325)] commit (INDIGO Sprint 230623, [!37](https://github.com/TeskaLabs/seacat-auth-webui/pull/37))
```

Comment:
hmmms.. is this really the latest master version of asab-webui submodule?...  🤔

Reply:
@queuing4oranges Petr is right, please update the CHANGELOG and also asab webui submodule to this commit:
https://github.com/TeskaLabs/asab-webui/commit/06af558428a2106e14d00d3fdb0b9457b761f787